pages/table/inline-editor.permutations.page.tsx

```diff
@@ -8,7 +8,7 @@ import Multiselect from '~components/multiselect';
 import createPermutations from '../utils/permutations';
 import PermutationsView from '../utils/permutations-view';
 import ScreenshotArea from '../utils/screenshot-area';
-
+const ref = React.createRef<any>();
```

Comment:
What is the purpose of this ref?

Reply:
I want to have it as a required prop in the component, so I needed to add here on the test page.